packages/terra-application-utility/src/terra-dev-site/test/application-utility/DefaultMenuUtilityMenu.test.scss

```diff
@@ -0,0 +1,15 @@
+@import './ApplicationUtilityTestCommon';
+
+:local {
+  .application-utility-test-menu-custom-component {
+    @extend %application-utility-test-custom-component;
```

Comment:
nit: could you use `application-utility-test-custom-component` directly if `application-utility-test-menu-custom-component` and `application-utility-test-custom-component` are no different?

Reply:
"DefaultMenuUtilityMenu.test.scss" and "DefaultHeaderUtilityMenu.test.scss" were identical in every manner. So, renamed one file to "UtilityMenuCommon.scss" and deleted the other. Also changed the references for the respective files.
I avoided using the classes in "_ApplicationUtilityTestCommon.scss" directly since they are meant only for extending/inheriting.